assess_birdnet/tools/normalize_scored_output.py

```diff
@@ -71,8 +71,18 @@ def mark_intervals(row, chunks_df):
     start_chunk = int(start_time // 3)
     end_chunk = int(end_time // 3)
 
-    if row['TOP1MATCH'] != 'null':
-        chunks_df.loc[start_chunk:end_chunk, 'Label'] = 'yes'
+    row_lower['MANUAL ID*'] = row['MANUAL ID*'].str.lower()
+
+    if row_lower['MANUAL ID*'] == 'cluck':
+        chunks_df.loc[start_chunk:end_chunk, 'Label'] = '0'
```

Comment:
Is this intended? This is going to assign multiple rows to a given label

<img width="528" height="456" alt="image" src="https://github.com/user-attachments/assets/3032e547-d013-456c-b735-b897610d484a" />